module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -473,33 +517,49 @@ namespace module::behaviour::strategy {
     }
 
     void SoccerStrategy::goalieWalk(const Field& field, const Ball& ball) {
```

Comment:
Is goalieWalk even being used since the goalie logic is currently to not move?

Reply:
#554 looks at this in depth

Reply:
I was wrong. Good point @abihall 
This function is used tho